client/pages/home.js

```diff
@@ -35,8 +35,15 @@ module.exports = function (state, emit) {
             A distributed <span class="color-green">dat</span>a community
           </h1>
           <h2 class="f3 mt3 mb3 color-neutral-50 fw2">
-            Dat is a nonprofit-backed community & open protocol for building apps of the future.
```

Comment:
I like having community in there but guess it makes sense to remove as its a big sentence!